tests/test_helpers.py

```diff
@@ -57,23 +57,17 @@
      ""
      ),
     (["helper", "u_milestone", "20sb_firstprojectum", "--number", "4",
-      "-s", "c", "--due_date", "2022-01-01"],
+      "-s", "c", "--due_date", "2020-05-30"],
      "20sb_firstprojectum has been updated in projecta\n"
      ),
     (["helper", "u_milestone", "20sb_firstprojectum"],
      "Please choose from one of the following to update/add:\n"
-     "1. Kick off meeting    due date: 2020-05-06    finished\n"
+     "1. new milestone\n"
      "2. Kick off meeting    due date: 2020-05-06    finished\n"
      "3. Project lead presentation    due date: 2020-05-20    proposed\n"
-     "4. deliverable    due date: 2021-05-05    finalized\n"
-     "5. planning meeting    due date: 2022-01-01    converged\n"
-     "6. new milestone\n"
+     "4. planning meeting    due date: 2020-05-30    converged\n"
+     "5. deliverable    due date: 2021-05-05    finalized\n"
      ),
-    # # this works! adding a new milestone:
```

Comment:
because the first test just lists, it doesn't update, we can actually have a second test that updates I think.  Please try.

Reply:
Thanks, professor!
The test in line 59 is updating the milestone number 4 (due date and status), so there are two tests, and the test on line 63 is listing. Do you mean adding one more? 
I will test it in rg-db-group! I would be happy to roll it out in the group meeting!
Should I be worried about the failing test on CI? It is passing locally, and I think it is the lister that is failing here.